src/main/java/seedu/bankwithus/storage/TransactionDecoder.java

```diff
@@ -0,0 +1,23 @@
+package seedu.bankwithus.storage;
+
+import seedu.bankwithus.user.Transaction;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+public class TransactionDecoder {
+
+    public static Transaction decodeTransaction(String encodedTransaction) {
+        String[] split = encodedTransaction.split(" /");
```

Comment:
Make sure you know the difference between splitting using " /" and splitting using "/". The second one feels less buggy imo

Reply:
Erm ok I think i know why you split using a " /" already. Why not use ";" instead? We have been using that as the separator since a while ago, and now you don't need the weird space before the slash

Reply:
OK